client/components/map/index.jsx

```diff
@@ -32,6 +32,7 @@ const map = ({
     zoom={zoom}
     onLeafletMoveend={(ev) => actions.map.getGeoJSON(ev)}
     onLeafletResize={(ev) => actions.map.getGeoJSON(ev)}
+    onLeafletClick={() => actions.map.setActive(undefined)}
```

Comment:
Without this there is no way to clear active trails. This means you could have a popup closed, then refresh and it would pull you back to that trail, and open the popup